.eslintrc

```diff
@@ -9,6 +9,7 @@
   "rules": {
     "quotes": [2, "single"],
     "eol-last": [0],
+    "consistent-return": [0],
```

Comment:
I might have to turn this back on, not sure if its giving me a legit warning or not...